fem/docker-compose-fem.yml

```diff
@@ -4,21 +4,21 @@ networks:
   monitoring:
 
 volumes:
-  fem_grafana_data:
```

Comment:
Remove the prefix and make use of the project prefix? I suppose the volume name at the end is like, "fem_grafana_data", right? What about the host- and container names? Should they changed as well?
Let's talk about it, what do you think?

Reply:
Ok, I thought about that point a bit. Container-, Volume- and Network-Name now all have prefix fem_ with only underscores. 
xyz.localhost dns resolution used the container_name which should be kept as fem_xyz. So I defined an extra_hosts entry which is more readable I suppose.
Maybe the hostname can be removed in future version.